### ISSUE TITLE
Doubutsu Banchou does not start with New Dynarec

### ISSUE THREAD

**Issue by loganmc10** [mupen64plus/mupen64plus-core#905]
Doubutsu Banchou, an unreleased game which became Cubivore for Gamecube, doesn't work with the new dynarec. https://archive.org/details/doubutsu-banchou-unreleased-carrot

It seems to work fine with the Cached Interpreter. I don't see any error messages in the console from the new dynarec, it just sits at a black screen

@Gillou68310

**Comment by m4xw** [comment]
This does work on my end with new_dynarec on aarch64/switch according to a tester

**Comment by Gillou68310** [comment]
@loganmc10 I pushed a fix for Doubutsu Banchou here :
https://github.com/Gillou68310/mupen64plus-core/tree/slt_sltu

Could you check if it fixes your issue?

**Comment by loganmc10** [comment]
@Gillou68310 just tried, didn't seem to make any difference. Tested on Ubuntu 20.04 x64, parallel rdp video. Confirmed that it did boot with the cached interpreter.

**Comment by Gillou68310** [comment]
Ok weird it works on win x64 too, linux x64 specific issue maybe?

**Comment by loganmc10** [comment]
I just tried it on Windows and got the same result, although it is compiled with msys2/mingw and not Visual Studio

**Comment by Gillou68310** [comment]
Yeah NVM I was testing with all interpreted fallback defined. Defining interpret_MULT is what fixes the issue. Looks like the issue is MULT in a branch delay slot, needs more investigation.
